src/borg/testsuite/archiver/create_cmd.py

```diff
@@ -243,9 +243,19 @@ def test_create_paths_from_command(self):
         self.create_regular_file("file4", size=1024 * 80)
 
         input_data = "input/file1\ninput/file2\ninput/file3"
-        self.cmd(
-            f"--repo={self.repository_location}", "create", "--paths-from-command", "test", "--", "echo", input_data
-        )
+        if is_win32:
+            with open("filenames.bat", "w") as script:
```

Comment:
.bat works, but is somehow ancient as it stems from DOS.

guess nowadays we rather have .cmd ?